lib.ts

```
@@ -0,0 +1,16 @@
+export const newAnchorage = (width: number, height: number): number[][] => {
+  if (width <= 0 || height <= 0) {
+    return [];
```

I would throw an error in case input size is invalid.